# Statistics Basic Assignment

### Q1. What are the three measures of central tendency?

Mean: The mean is often referred to as the "average." It is calculated by summing up all the values in the dataset and then dividing by the total number of values. The mean can be affected by extreme values, also known as outliers, since it considers all values equally.

Median: The median is the middle value in a dataset when the data is arranged in order. If there is an odd number of values, the median is the middle value itself. If there's an even number of values, the median is the average of the two middle values. The median is less sensitive to outliers compared to the mean, making it a better measure of central tendency when the data contains extreme values.

Mode: The mode is the value that appears most frequently in a dataset. A dataset can have one mode (unimodal), two modes (bimodal), or more (multimodal). The mode can be particularly useful for categorical or discrete data, but it's also applicable to continuous data.

### Q2. What is the difference between the mean, median, and mode? How are they used to measure the central tendency of a dataset?

Mean:

The mean is the arithmetic average of all the values in a dataset.
It is calculated by adding up all the values and then dividing by the total number of values.
The mean can be sensitive to extreme values (outliers) that can significantly affect its value.
It's commonly used when the data has a symmetrical distribution, such as the normal distribution.
Median:

The median is the middle value of a dataset when it's arranged in order.
If there's an odd number of values, the median is the middle value itself.
If there's an even number of values, the median is the average of the two middle values.
The median is less affected by extreme values and outliers compared to the mean, making it a good choice for skewed data or when you want to mitigate the influence of outliers.
Mode:

The mode is the value that appears most frequently in a dataset.
A dataset can have one mode (unimodal), two modes (bimodal), or more (multimodal).
The mode is particularly useful for categorical or discrete data, where the concept of middle value might not apply.
It's often used to describe the most common outcome or observation in a dataset.
In summary, these measures capture different aspects of the central tendency in a dataset:

Use the mean when you want a balanced measure that considers all values equally and when the data is relatively symmetric.

Use the median when you want a measure that's less sensitive to outliers and when the data is skewed or contains extreme values.

Use the mode when you want to identify the most frequent value or category in the data.

### Q3. Measure the three measures of central tendency for the given height data:
[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [1]:
h_data = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [2]:
import numpy as np
np.mean(h_data)

177.01875

In [3]:
np.median(h_data)

177.0

In [4]:
import statistics
statistics.mode(h_data)

178

### Q4. Find the standard deviation for the given data:

In [5]:
data = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [6]:
np.std(data)

1.7885814036548633

### Q5. How are measures of dispersion such as range, variance, and standard deviation used to describe the spread of a dataset? Provide an example.


Measures of dispersion are used to describe the spread of a dataset. They measure how far apart the data points are from each other. The most common measures of dispersion are:

Range: The range is the difference between the largest and smallest values in the dataset. It is the simplest measure of dispersion, but it is not very sensitive to outliers.

Variance: The variance is the average squared deviation from the mean. It is a more robust measure of dispersion than the range, but it is also more difficult to interpret.

Standard deviation: The standard deviation is the square root of the variance. It is the most commonly used measure of dispersion because it is easy to interpret and it is relatively insensitive to outliers.

Here is an example. Let's say we have the following data set:

1, 2, 3, 4, 5, 6, 7, 8, 9, 10

The range of this data set is 9 (10 - 1). The variance is 3.29 (sum of (each data point - mean)^2 / 10). The standard deviation is 1.8 (square root of the variance).

The range tells us that the data points in this set are spread out over a range of 9 values. The variance tells us that the data points are, on average, 3.29 away from the mean. The standard deviation tells us that the data points are, on average, 1.8 away from the mean.

In general, a larger value of any of these measures of dispersion indicates that the data points are more spread out. A smaller value indicates that the data points are more tightly clustered together.

### Q6. What is a Venn diagram?

A Venn diagram is a visual representation used to illustrate the relationships between different sets or groups of items. It consists of overlapping circles (or sometimes other shapes) that are used to represent these sets, and the overlapping regions show the intersections or common elements between the sets.

The purpose of a Venn diagram is to provide a clear and intuitive way to understand the relationships and commonalities between different sets of data or concepts. Each circle in the diagram represents a separate set, and the areas where the circles overlap represent the elements that belong to both sets. The areas that do not overlap represent the elements unique to each set.

### Q7. For the two given sets A = (2,3,4,5,6,7) & B = (0,2,6,8,10). Find:
(i) A ∩ B
(ii) A ⋃ B

A ∩ B = {2}

A ⋃ B = {0,2,3,4,5,6,7,8,10}

### Q8. What do you understand about skewness in data?

Skewness is a statistical term that describes the asymmetry or lack of symmetry in the distribution of data points within a dataset. It is an important measure in statistics and data analysis as it helps to understand the shape of the data distribution.

There are three main types of skewness:

Positive Skewness (Right Skewness):
In a positively skewed distribution, the tail on the right side is longer or stretched out, while the bulk of the data points is concentrated on the left side. This indicates that the majority of the data points are smaller, and there are a few large values that pull the mean in the positive direction.

Negative Skewness (Left Skewness):
In a negatively skewed distribution, the tail on the left side is longer or stretched out, while the majority of data points are concentrated on the right side. This suggests that most of the data points are larger, and there are a few smaller values that pull the mean in the negative direction.

No Skewness (Symmetrical Distribution):
A symmetrical distribution has equal tails on both sides of the central peak, and the data points are distributed evenly around the mean. This results in a balanced distribution without any skewness.

### Q9. If a data is right skewed then what will be the position of median with respect to mean?

In a right skewed distribution, the mean is greater than the median. This is because the mean is affected by outliers, which are data points that are much larger or smaller than the rest of the data. Outliers can pull the mean towards the tail of the distribution, making it larger than the median.

The median, on the other hand, is not affected by outliers. It is the middle value of the data set, when all the data is ranked from least to greatest. So, in a right skewed distribution, the median will be located to the left of the mean.

### Q10. Explain the difference between covariance and correlation. How are these measures used in statistical analysis?

Covariance and correlation are both measures used in statistics to quantify the relationship between two variables. They provide insights into how changes in one variable are associated with changes in another variable. While they are related concepts, they have distinct characteristics and purposes.

Covariance:
Covariance measures the extent to which two variables change together. It indicates the direction of the linear relationship between the variables. A positive covariance suggests that when one variable increases, the other variable tends to increase as well. A negative covariance suggests that when one variable increases, the other variable tends to decrease. However, covariance doesn't provide a standardized measure of the strength of the relationship and is sensitive to the units of measurement of the variables.

Correlation:
Correlation measures the strength and direction of the linear relationship between two variables, and it's a standardized measure that ranges between -1 and 1. A correlation of +1 indicates a perfect positive linear relationship, a correlation of -1 indicates a perfect negative linear relationship, and a correlation close to 0 suggests a weak or no linear relationship. Correlation is unitless and helps compare the strength of relationships between variables with different units.

Usage in Statistical Analysis:

Covariance: Covariance is used to identify whether changes in one variable tend to occur at the same time as changes in another variable. It's used in portfolio management, finance, and risk assessment to analyze relationships between different assets' returns.

Correlation: Correlation is widely used to quantify the strength of relationships between variables. It's used in fields such as economics, social sciences, and engineering to study cause-and-effect relationships, make predictions, and assess the quality of data. Correlation is also a key concept in data analysis and regression modeling.

### Q11. What is the formula for calculating the sample mean? Provide an example calculation for a dataset.

X̄ = (Σ xi) / n

# Example:
Consider a dataset of exam scores: 85, 90, 78, 92, 88.

To calculate the sample mean:

Add up all the individual data points: 
85+90+78+92+88=433

Count the number of data points: 
n=5.
Divide the sum by the number of data points: 
433/5
86.6
So, the sample mean for this dataset is 
86.6.

### Q12. For a normal distribution data what is the relationship between its measure of central tendency?

In a normal distribution, all three measures of central tendency are equal: the mean, median, and mode. This is because a normal distribution is symmetrical, with the mean, median, and mode all located at the center of the distribution.

The mean is the average of all the values in the distribution. The median is the middle value, when all the values are ranked from least to greatest. The mode is the most frequent value in the distribution.

In a normal distribution, the mean, median, and mode are all equal because the distribution is symmetrical. This means that there are just as many values above the mean as there are below the mean. There are also just as many values above the median as there are below the median. And there is just one value that appears more often than any other value.

### Q13. How is covariance different from correlation?

Covariance and correlation are related concepts that both describe the relationship between two variables, but they have different characteristics and interpretations. Here's how they differ:

1. Definition:

Covariance: Covariance measures the degree to which two variables change together. It indicates whether the variables tend to increase or decrease at the same time. A positive covariance suggests a positive relationship, while a negative covariance suggests a negative relationship.
Correlation: Correlation measures the strength and direction of the linear relationship between two variables. It provides a standardized measure that ranges between -1 (perfect negative correlation) and 1 (perfect positive correlation).

2. Units:

Covariance: Covariance is not standardized and is affected by the units of the variables. As a result, it's not directly comparable across different datasets or variables.
Correlation: Correlation is unitless and is not affected by the units of the variables. This makes it suitable for comparing relationships between variables with different units.

3. Range:

Covariance: Covariance values have no specific range, which can make it difficult to interpret the strength of the relationship.
Correlation: Correlation values range between -1 and 1. A correlation of +1 indicates a perfect positive linear relationship, -1 indicates a perfect negative linear relationship, and 0 indicates no linear relationship.

4. Interpretation:

Covariance: Covariance alone doesn't provide a clear indication of the strength of the relationship or whether it's significant.
Correlation: Correlation not only indicates the direction but also provides a measure of the strength of the linear relationship between the variables. It's easier to interpret: a correlation close to 1 or -1 indicates a strong linear relationship, while a correlation close to 0 indicates a weak or no linear relationship.

5. Standardization:

Covariance: Covariance isn't standardized, so its magnitude isn't readily interpretable.
Correlation: Correlation is standardized and provides a clear indication of the strength of the linear relationship relative to the variability of the variables.

### Q14. How do outliers affect measures of central tendency and dispersion? Provide an example.

Outliers are data points that deviate significantly from the rest of the data in a dataset. They can have a notable impact on measures of central tendency (such as mean, median, and mode) and measures of dispersion (such as range, variance, and standard deviation). Here's how outliers affect these measures and an example to illustrate:

Impact on Measures of Central Tendency:

Mean: Outliers can strongly influence the mean because they contribute to the sum of values. A single outlier that is much larger or smaller than the rest of the data can pull the mean in the direction of the outlier.

Median: The median is less sensitive to outliers because it's not directly affected by the actual values of the data. However, extreme outliers can still affect the position of the median, especially in small datasets.

Mode: The mode may not be greatly affected by outliers, as it's determined by the most frequent value(s) in the data.

Impact on Measures of Dispersion:

Range: Outliers can significantly increase the range if they are far from the rest of the data points.

Variance and Standard Deviation: Outliers can substantially inflate the variance and standard deviation. This is because these measures involve squaring the deviations from the mean, and large deviations (due to outliers) result in even larger squared deviations.

Example:

Consider a dataset of exam scores: 85, 87, 88, 92, 94, 96, 100, 102, 110, 250.

In this example, the value "250" is an outlier, as it is much larger than the other scores. Let's see how the outliers affect measures of central tendency and dispersion:

Mean: The mean without the outlier is 
(85+87+88+92+94+96+100+102+110)/9≈95.44. With the outlier, the mean becomes 

(85+87+88+92+94+96+100+102+110+250)/10≈102.4.
Median: The median without the outlier is the middle value, which is 96. With the outlier, the median remains the same because it's not directly affected by the magnitude of the values.
Range: Without the outlier, the range is 
110−85=25. With the outlier, the range becomes 
165
250−85=165.
Standard Deviation: Without the outlier, the standard deviation is approximately 7.52. With the outlier, the standard deviation increases to approximately 49.97.